r/src/Makevars.win

```diff
@@ -1,38 +0,0 @@
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-
-VERSION = 0.13.0.9000
```

Comment:
@kou did something go wrong, I'd expect the version to be `0.14.0.9000`. This is the values Rust and Java uses for example.

Reply:
0.13.0.9000 is "correct" (here at least) for the dev version after 0.13 before 0.14.

Reply:
Ah, so the inverse of Java?

Reply:
Apparently so? IDK, the version math makes more sense to me the way that this is done in the R community, `0.13.0 < 0.13.0.9000 < 0.14.0`. I'm not familiar with how Java handles this.

Reply:
`0.13.0.9000` is correct.

See also: https://issues.apache.org/jira/browse/ARROW-4113?focusedCommentId=16728608&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-16728608

Rust and Java use `0.14.0-SNAPSHOT`. They don't use `0.14.0.9000`.

We use `X.Y.Z-SNAPSHOT` for languages that support PRE release by tag ("SNAPSHOT").
R doesn't support tag. So we use ".9000". ".9000" is commonly used in R.

Reply:
cf. http://r-pkgs.had.co.nz/release.html for the R convention.